sdk/translation/ai-translation-document-rest/api-extractor.json

```diff
@@ -11,7 +11,7 @@
   "dtsRollup": {
     "enabled": true,
     "untrimmedFilePath": "",
-    "publicTrimmedFilePath": "./types/ai-translation-document-rest.d.ts"
+    "publicTrimmedFilePath": "./types/ai-translation-document.d.ts"
```

Comment:
is this generated?  maybe the files in package.json needs to be update to include -rest instead? https://github.com/Azure/azure-sdk-for-js/blob/c2b68a78efde436d5b39f5af38af224912c0e6d3/sdk/translation/ai-translation-document-rest/package.json#L29

If this entry is also generated, then there is an issue in codegen.

Reply:
I suggested updating to remove the `-rest` since this is consistent with the other generated RLCs ([example](https://github.com/Azure/azure-sdk-for-js/pull/17517#issuecomment-918635850)). I was wondering if there was a codegen bug too but I am slightly perplexed since at a glance the same value is used in the [package.json files section](https://github.com/Azure/autorest.typescript/blob/main/packages/rlc-common/src/metadata/packageJson/azurePackageCommon.ts#L77), [api extractor config](https://github.com/Azure/autorest.typescript/blob/main/packages/rlc-common/src/metadata/buildApiExtractorConfig.ts#L35), and [package.json types entry](https://github.com/Azure/autorest.typescript/blob/main/packages/rlc-common/src/metadata/packageJson/packageCommon.ts#L83). @hamshavathimunibyraiah, do you recall having to make any manual changes to any of these files to get things to work?

Reply:
for those metadata files, we will only generate them when there's no such files exist or generateMetadata: true is enabled, if we generate the code with no `-rest` as the package name for the first time, and rename it later and never enabled `generateMetadata: true` when regen the code, cases like that could happen.

Reply:
as long as this is not a codegen issue, it should be fine.